### DISCUSSION TITLE
请问 Maple UI 有计划更新吗？

### DISCUSSION THREAD

**Discussion by MiroKaku** [subframe7536/maple-font#302] [Q&A]
Maple UI 在电脑上作为全局字体效果很好，但是字符集不完整，请问后面会包含更多的字符字体吗？我发现最新的7.0版本也没有包含这些字符。

![图片](https://github.com/user-attachments/assets/1f82ce97-093a-44a4-9d13-704fd25330fc)

上图来自使用 Maple 字体后的 [Microsoft Docs](https://learn.microsoft.com/zh-cn/)

**Comment by subframe7536** [comment]
目前暂无计划，我现在主要用的是 MiSans
